### PR TITLE
View all tags

### PR DESCRIPTION
**Pull request by tzufgoogle**
The web application is SPA.

We added:
1. Added all tags to view on the map.
2. Added 2 main pivots (a total of 3 main pivots).
3. Added 3 pivots around the goal but not near it.
4. The pivots near the goal do not show the name tag.
5. Added data v6-v24 with these changes.

### CODE REVIEW

File: cabby/geo/visualization/database.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+
+from datetime import datetime
+from flask import *
```
Comment (jasonbaldridge) [inline cabby/geo/visualization/database.py:3]: Don't import all from a module. Instead 'import flask' and then use flax.foo, etc., or use 'from flax import foo' for each thing you need.

File: cabby/geo/visualization/forms.py [hunk 2]
```diff
@@ -12,32 +14,36 @@ def check_no_first_person(form, field):
...
+# def check_verbs(form, field):
+#   # word tokenizeing and part-of-speech tagger
+#   tokens = [nltk.word_tokenize(sent) for sent in [field.data]]
+#   postag = [nltk.pos_tag(sent) for sent in tokens][0]
+#   tags = [pt[1] for pt in postag if 'VB' in pt[1]]
 
-  if len(tags)<1:
-    raise ValidationError('Try again to describe the goal.')
+#   if len(tags)<1:
+#     raise ValidationError('Try again to describe the goal.')
```
Comment (jasonbaldridge) [inline cabby/geo/visualization/forms.py:24]: Can delete commented out code.

File: cabby/geo/visualization/templates/follower_task.html [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+
+        
+<div id="mapid" class=map></div>
+<script>
+
+    var map = L.map('mapid').setView({{start_point|tojson}}, 16);
+
+
+	L.tileLayer('https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}?access_token=pk.eyJ1IjoibWFwYm94IiwiYSI6ImNpejY4NXVycTA2emYycXBndHRqcmZ3N3gifQ.rJcFIG214AriISLbB6B5aw', {
```
Comment (jasonbaldridge) [inline cabby/geo/visualization/templates/follower_task.html:28]: Is this a private access token that shouldn't be checked into to OS code? (If so, need to regenerate and pass in another way.)

File: cabby/geo/walk.py [hunk 4]
```diff
@@ -42,20 +42,23 @@
...
 # The max number of failed tries to generate a single path entities.
 MAX_NUM_GEN_FAILED = 10
 PIVOT_ALONG_ROUTE_MAX_DIST = 0.0001
 ADD_POI_DISTANCE = 5000
 MAX_NUM_BEYOND_TRY = 50
 
 LANDMARK_TYPES = [
-  "end_point", "start_point", "main_pivot", "near_pivot", "beyond_pivot"]
+  "end_point", "start_point", "main_pivot", "main_pivot_2", "main_pivot_3", "near_pivot",
+   "beyond_pivot", "around_goal_pivot_1", "around_goal_pivot_2", "around_goal_pivot_3"]
```
Comment (jasonbaldridge) [inline cabby/geo/walk.py:61]: This was defined in util.py -- why have it in two spots?

**Review by jasonbaldridge — CHANGES_REQUESTED** [review]
Thanks! Quite a large PR. Couldn't review in very fine detail, but found a number of things to fix and/or consider.

**Review by tzufgoogle — COMMENTED** [review]
1. Changed NOT_DISPLAY_TAGS name 
2. Removed SECRET_KEY

**Review by jasonbaldridge — APPROVED** [review]
Thanks!

**Pull request merged**